engine/rotationmanager/queries.sql

```diff
@@ -0,0 +1,79 @@
+-- name: RotMgrRotationData :one
+-- Get rotation data for a given rotation ID
+SELECT
+    now()::timestamptz AS now,
+    sqlc.embed(rot),
+    coalesce(state.version, 0) AS state_version,
+    coalesce(state.position, 0) AS state_position,
+    state.shift_start AS state_shift_start,
+    ARRAY (
+        SELECT
+            p.id
+        FROM
+            rotation_participants p
+        WHERE
+            p.rotation_id = rot.id
+        ORDER BY
+            position)::uuid[] AS participants
```

Comment:
has this already been linted?
```suggestion
            position
      )::uuid[] AS participants
```

Reply:
it has to have the cast so that `sqlc` infers the correct type -- it's usually pretty good but any coalesce or nested queries (like this one) tend to cause trouble without an explicit cast